tests/foreman/destructive/test_remoteexecution.py

```diff
@@ -58,16 +59,20 @@ def test_negative_run_capsule_upgrade_playbook_on_satellite(target_sat):
                     'whitelist_options': "repositories-validqqate,repositories-setup",
                 },
                 'targeting_type': "static_query",
-                'search_query': f"name = {sat.name}",
+                'search_query': f"name = {target_sat.hostname}",
             }
         )
     assert 'A sub task failed' in error.value.args[0]
     job = target_sat.api.JobInvocation().search(
-        query={'search': f'host={sat.name},status=failed,description="Capsule Upgrade Playbook"'}
+        query={
+            'search': f'host={target_sat.hostname},'
+            'status=failed,description="Capsule Upgrade Playbook"'
+        }
     )[0]
+    host = target_sat.api.Host().search(query={'search': target_sat.hostname})
     response = client.get(
-        f'{target_sat.url}/api/job_invocations/{job.id}/hosts/{sat.id}',
-        auth=(target_sat.username, target_sat.password),
+        f'{target_sat.url}/api/job_invocations/{job.id}/hosts/{host[0].id}',
```

Comment:
using the following patch, you can delete L72.
```suggestion
        f'{target_sat.url}/api/job_invocations/{job.id}/hosts/{target_sat.nailgun_host.id}',
```

Reply:
hm, for some reason this didn't work for me when I was trying that out locally